packages/rust-bindings/packages/wallet/Cargo.toml

```diff
@@ -15,15 +15,16 @@ hex.workspace = true
 either.workspace = true
 thiserror.workspace = true
 rand.workspace = true
+getrandom.workspace = true
 serde-wasm-bindgen.workspace = true
 concordium_base.workspace = true
 ed25519_hd_key_derivation.workspace = true
 key_derivation.workspace = true
 wallet_library.workspace = true
 concordium_rust_bindings_common.workspace = true
-ed25519-dalek = { version = "=1.0" }
-chrono = "0.4.24"
-serde_with = "3.0.0"
+ed25519-dalek = { version = "2.0" }
```

Comment:
Cannot create a `KeyPair` without the `SigningKey` type from this crate, but I think it would make sense to export the `SigningKey` type from `concordium-base`. Will add that to https://github.com/Concordium/concordium-base/pull/531